packages/pricing-tier-list/README.md

```diff
@@ -0,0 +1,3 @@
+# Pricing Tier List
+
+## @hashicorp/react-pricing-tier-list
```

Comment:
A short description here would be great! Even just one or two sentences 👍

Reply:
👍  Added [here](https://github.com/hashicorp/react-components/pull/644/commits/b2138260c7d8ced2a74b1515b36694b8c6c7f63b). Open to suggestions tho